x/fundraising/keeper/vesting.go

```diff
@@ -29,20 +29,28 @@ func (k Keeper) SetVestingSchedules(ctx sdk.Context, auction types.AuctionI) err
 		k.SetAuction(ctx, auction)
 
 	} else {
-		for _, vs := range auction.GetVestingSchedules() {
-			payingAmt := reserveBalance.Amount.ToDec().Mul(vs.Weight).TruncateInt()
+		if err := k.bankKeeper.SendCoins(ctx, payingReserveAcc, vestingReserveAcc, reserveCoins); err != nil {
+			return err
+		}
+
+		remaining := reserveCoin
+		for i, vs := range auction.GetVestingSchedules() {
+			payingAmt := reserveCoin.Amount.ToDec().MulTruncate(vs.Weight).TruncateInt()
+
+			// store remaining to the paying coin in the last queue
+			if i == len(auction.GetVestingSchedules())-1 {
```

Comment:
`len(auction.GetVestingSchedules())-1` could exist as a variable outside the for loop so that the same `len()` may not be repeatedly performed every time like below.

`if i == len(lenVestingSchedules)-1 {`

Reply:
You're right. I will take that out of the for loop.